src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -1130,6 +1159,14 @@ public String constructManifestAndUserData(boolean graphAlreadyBuilt, boolean dr
         return String.join("\n", lines);
     }
 
+    /**
+     * Return the appropriate otp-runner manifest filename. Two different manifest files are generated when deploying
+     * with different instance types for graph building vs server running.
+     */
+    private String getOtpRunnerManifestS3Filename(boolean graphAlreadyBuilt) {
+        return graphAlreadyBuilt ? "otp-runner-server-manifest.json" : "otp-runner-graph-build-manifest.json";
```

Comment:
I kinda prefer the file names you used for the test snapshots, and here they would be
`otp-runner-server-only-manifest.json` and `otp-runner-build-and-serve-manifest.json` resp.

Reply:
That problem with this naming is that sometimes the non-server-only manifest will be for either build-only or build-and-serve. I went ahead and renamed the build file.